aspnetcore/fundamentals/minimal-apis.md

```diff
@@ -1555,19 +1555,44 @@ Routes can be [CORS](xref:security/cors?view=aspnetcore-6.0) enabled using [CORS
 
 For more information, see <xref:security/cors?view=aspnetcore-6.0>
 
+<a name="openapi7"></a>
+
 ## OpenAPI
 
 An app can describe the [OpenAPI specification](https://swagger.io/specification/) for route handlers using [Swashbuckle](https://www.nuget.org/packages/Swashbuckle.AspNetCore/).
 
-The following code is a typical ASP.NET Core app with OpenAPI support:
+The following code is generated by the ASP.NET Core minimal web API template and uses OpenAPI:
+
+[!code-csharp[](minimal-apis/7.0-samples/WebMinOpenApi/Program.cs?name=snippet&highlight=1,5-6,12-13,35-36)]
+
+The OpenAPI specification defines a standard for documenting RESTful APIs so that end users and other services can examine the capabilities of an API service.
```

Comment:
Reading this on the published site it reads super strange seeing the generated template code and then diving into this explanation. Could we either:

(a) remove the template code sample above
(b) add some sort of transition sentence between the code sample and the content on the package